generative_models/3D-GAN_PyTorch/README.md

```diff
@@ -0,0 +1,117 @@
+# **PyTorch implementation of 3D-GAN**
+## Usage
+```bash
+> python main.py --arguments
+```
```

Comment:
add script to download 'modelnet10.npy.gz' something like `wget <download link>`

Reply:
Yes I will do that, but that does not work. I mean the downloaded file shows some error when we try to unzip using gzip on python. 

If you want, I will still add the link ?